lib/memo_wise.rb

```diff
@@ -190,7 +190,7 @@ def #{method_name}#{args_str}
         end
 
         module_eval <<-END_OF_VISIBILITY, __FILE__, __LINE__ + 1
-          "#{method_visibility} :#{method_name}"
+          #{method_visibility} :#{method_name}
```

Comment:
We didn't need the `""` here because the `END_OF_VISIBILITY` tags are wrapping this in a string

Reply:
Happy to keep this as-is, but would it make sense to just use a regular string instead of a heredoc here?